fluentd/configs.d/openshift/es-copy-config.conf

```diff
@@ -28,4 +28,5 @@
       # queue limit is hit - 'block' will halt further reads and keep retrying to flush the
       # buffer to the remote - default is 'exception' because in_tail handles that case
       buffer_queue_full_action "#{ENV['BUFFER_QUEUE_FULL_ACTION'] || 'exception'}"
+      ssl_verify "#{ENV['ES_COPY_SSL_VERIFY']}"
```

Comment:
This is being added for the CI test test-es-copy.sh, in which the copy output has to have an elasticsearch pod having the different hostname and/or port from the primary elasticsearch to forward the copy (otherwise, this PR would skip configuring the copy output).  Please note that in the existing test-es-copy.sh, the copy output is forwarding to the same elasticsearch and check the elasticsearch has duplicated logs.  To use the same logic in test-es-copy.sh, we decided to use HOSTNAME-copy for the alias hostname for the target of copy output.  But forwarding failed due to the hostname mismatch in the cert.  To work around it, @richm game me this advice to set "ssl_verify false".